lsp-mode.el

```diff
@@ -4236,6 +3932,8 @@ in that particular folder."
       (lsp--semantic-tokens-initialize-buffer
        (lsp-feature? "textDocument/semanticTokensRangeProvider"))))
 
+  (run-hooks 'lsp-configure-hook)
```

Comment:
Shouldn't you move this inside `(when lsp-auto-configure)` block? Else when new dynamic registration changes come, breadcrumb will be enabled even without `lsp-auto-configure`.

Reply:
Yes, done!